connector/connect/common/src/main/protobuf/spark/connect/commands.proto

```diff
@@ -484,3 +486,26 @@ message CreateResourceProfileCommandResult {
   // (Required) Server-side generated resource profile id.
   int32 profile_id = 1;
 }
+
+// Command to remove `CashedRemoteRelation`
+message RemoveCachedRemoteRelationCommand {
+  // (Required) The remote to be related
+  CachedRemoteRelation relation = 1;
+}
+
+message CheckpointCommand {
+  // (Required) The logical plan to checkpoint.
+  Relation relation = 1;
+
+  // (Optional) Locally checkpoint using a local temporary
+  // directory in Spark Connect server (Spark Driver)
+  optional bool local = 2;
+
+  // (Optional) Whether to checkpoint this dataframe immediately.
+  optional bool eager = 3;
```

Comment:
The default is `eager = true` right? Should the protocol encode this better? Currently the protocol defaults to `eager = false` if the field is not set, so my question is should we flip the logic (i.e. replace this with `lazy`) so the default behavior does not require you to set additional fields.

The same question for `local`...